frodo.js

```diff
@@ -1,65 +1,65 @@
 /**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?